odoo/addons/test_new_api/security/ir.model.access.csv

```diff
@@ -64,6 +64,8 @@ access_test_new_api_model_selection_base,access_test_new_api_model_selection_bas
 access_test_new_api_model_selection_required,access_test_new_api_model_selection_required,model_test_new_api_model_selection_required,,1,1,1,1
 access_test_new_api_model_selection_non_stored,access_test_new_api_model_selection_non_stored,model_test_new_api_model_selection_non_stored,,1,1,1,1
 access_test_new_api_model_selection_required_for_write_override,access_test_new_api_model_selection_required_for_write_override,model_test_new_api_model_selection_required_for_write_override,,1,1,1,1
+access_test_new_api_model_selection_related,access_test_new_api_model_selection_related,test_new_api.model_test_new_api_model_selection_related,base.group_user,1,0,0,0
+access_test_new_api_model_selection_related_updatable,access_test_new_api_model_selection_related_updatable,test_new_api.model_test_new_api_model_selection_related_updatable,base.group_user,1,0,0,0
```

Comment:
The tests do create and write on those models.  Making them readonly in the ACL is simply confusing, as it only works because the tests are done with the superuser.  Grant full permission as the models around it.